### PR TITLE
Add error handling to "New note from URL" dialog

### LINKED ISSUE #3308: Can't make new notes after failed URL to note conversion
# Current behavior

The new note button stops working after entering a non existent URL to convert.

# Expected behavior

A failure message appears about the previous URL to note conversion.
AND
The new note button does not stop working after a failure.

# Steps to reproduce

1. Open the new note selection (Ctrl+N).
2. Select new markdown note from URL.
3. Enter a non existent URL, e.g. https://www.bgvoirhbvoojwscj.com
4. Hit Enter. (No new note is made, as expected.)
4. Try clicking the new note button (or Ctrl+N).

Dev tools console: 
image

# Workaround

Reload Boostnote (Ctrl+R) to fix the issue temporarily.

# Environment

- Version : 0.13.0
- OS Version and name : Ubuntu 16.04

### PR DESCRIPTION
**Pull request by AWolf81**
## Description

Adds error handling for "New note from URL" dialog.

## Issue fixed
Fixes #3308

- Display error message on url not found
- Keep "New note" functional

## Type of changes

- :radio_button: Bug fix (Change that fixed an issue)
- :white_circle: Breaking change (Change that can cause existing functionality to change)
- :white_circle: Improvement (Change that improves the code. Maybe performance or development improvement)
- :white_circle: Feature (Change that adds new functionality)
- :white_circle: Documentation change (Change that modifies documentation. Maybe typo fixes)

## Checklist:

- :radio_button: My code follows [the project code style](docs/code_style.md)
- :white_circle: I have written test for my code and it has been tested
- :radio_button: All existing tests have been passed
- :radio_button: I have attached a screenshot/video to visualize my change if possible

## Screenshots
**URL not found**
grafik

**Wrong url format - missing .com**
grafik

## Comments
- Is it OK to add the messages as a constant into the file? Or is there a constants file where we could add it?
- `Unexpected` error message is just a fallback if the code is not matching and I think that's OK.
- Are the error messages OK? I've added the check your internet connection because you're also getting `ENOTFOUND` with-out internet connection.

### CODE REVIEW

File: browser/main/lib/dataApi/createNoteFromUrl.js [hunk 1]
```diff
@@ -14,12 +14,18 @@ function validateUrl (str) {
   }
 }
 
+const ERROR_MESSAGES = {
+  ENOTFOUND: 'URL not found. Please check the URL, your internet connection and try again.',
```
Comment (Flexo013) [inline browser/main/lib/dataApi/createNoteFromUrl.js]: I would rephrase this as:
`URL not found. Please check the URL, or your internet connection and try again.`

File: browser/main/lib/dataApi/createNoteFromUrl.js [hunk 2]
```diff
@@ -14,12 +14,18 @@ function validateUrl (str) {
...
+  VALIDATION_ERROR: 'Please check your URL is in correct format. (Example, https://www.google.com)',
```
Comment (Flexo013) [inline browser/main/lib/dataApi/createNoteFromUrl.js]: I would rephrase this as:
`Please check if the URL is in the correct format. (Example, https://www.google.com)`

File: browser/main/lib/dataApi/createNoteFromUrl.js (hunk 1 above)
Comment (AWolf81) [reply browser/main/lib/dataApi/createNoteFromUrl.js]: That's better. I've updated it.

File: browser/main/lib/dataApi/createNoteFromUrl.js (hunk 2 above)
Comment (AWolf81) [reply browser/main/lib/dataApi/createNoteFromUrl.js]: OK. I've changed it.
